src/py/flwr/server/superlink/fleet/grpc_rere/fleet_servicer.py

```diff
@@ -51,27 +51,29 @@ def CreateNode(
         self, request: CreateNodeRequest, context: grpc.ServicerContext
     ) -> CreateNodeResponse:
         """."""
-        log(INFO, "FleetServicer.CreateNode")
+        log(INFO, "FleetServicer: CreateNode %s", request)
```

Comment:
At the end of this method, adding the response is in a separate `DEBUG` log, should we do the same here?

Reply:
I'd also say that we can shorten it to `[Fleet.CreateNode]` and add this to every log coming from this message (same for all the other logs in the servicer):
```suggestion
        log(INFO, "[Fleet.CreateNode] Request %s", request)
```